ndbench-cass-plugins/src/main/java/com/netflix/ndbench/plugin/cass/CJavaDriverBasePlugin.java

```diff
@@ -57,6 +57,8 @@ public void init(DataGenerator dataGenerator) throws Exception {
         KeyspaceName = propertyFactory.getProperty(NdBenchConstants.PROP_NAMESPACE +"cass.keyspace").asString("dev1").get();
         TableName =propertyFactory.getProperty(NdBenchConstants.PROP_NAMESPACE +"cass.cfname").asString("emp").get();
         port = propertyFactory.getProperty(NdBenchConstants.PROP_NAMESPACE + "cass.host.port").asInteger(9042).get();
+        Replication = propertyFactory.getProperty(NdBenchConstants.PROP_NAMESPACE +"cass.replication").asString("{'class': 'NetworkTopologyStrategy','eu-west': '3','us-east': '3'}").get();
```

Comment:
Ideally kesypace and table creation should be done outside of NDBench as it is a not a best practice to creat from code, as you could have multiple NDBench instances running which are trying to create keyspace at same time, and C* is not great with handling concurrent schema modifications. The reason, we have a keyspace creation in code is as last resort, in case if you are playing with it. So I would try to avoid any encouragement which makes keyspaces and table creation easy from code until C* gets better at handling that.

Reply:
Thanks for pointing it out. But please note that I did not add the Keyspace there just added the option for, in the case it is needed to set the keyspace through code, to also be able to set the number of connections and replication factor. I hope you will welcome that.
Btw, what is the correct way to do the keyspace creation?

Reply:
Yes, number of connections are great. I am happy with that patch. Replication seems to be overkill, keyspace name has to be configurable since it can be user specific. I would create keyspace without any concurrent threads, may be manually or startup script in some automation tool, init scripts before starting the load.